core/modules/savers/git-service.js

```diff
@@ -17,26 +17,23 @@ var base64utf8 = require("$:/core/modules/utils/base64-utf8/base64-utf8.module.j
 /*
 Select the appropriate saver module and set it up
 */
-var GitHubSaver = function(wiki) {
+var GitServiceSaver = function(wiki) {
```

Comment:
Why not use class syntax? All major browser supports it now.

Reply:
Because of coding style consistency.

Reply:
Hi @linonetwo we intentionally stick to ES5 to give maximum compatibility; there are a lot of old JS environments out there. There are lots of reasons we don't use transpilation chiefly because browsers don't support source maps for single files like TW.